### PR TITLE
[RW-2594][risk=no] New research purpose on ws About page

### PR DESCRIPTION
**Pull request by jacmrob**

### CODE REVIEW

File: ui/src/app/components/tables.tsx [hunk 1]
```diff
@@ -114,3 +119,32 @@ export const TextCell = ({style = {}, ...props}) => {
...
 };
+
+export const PaddedTableCell = ({left, content, leftWidth = '50%', rightWidth = '50%'}) => {
+  return <div style={{padding: '.2rem', width: left ? leftWidth :
+      rightWidth, display: 'flex'}}>
+    <div style={{...styles.paddedTable, width: '100%'}}>{content}</div>
+  </div>;
+};
+
+export const TwoColPaddedTable = ({style = {}, header = false, headerLeft = '',
```
Comment (gjuggler) [inline ui/src/app/components/tables.tsx]: +1 on the effort to create a reusable component here – looks nice.

**Comment by jacmrob** [comment]
Screen Shot 2019-06-07 at 9 34 29 AM

File: ui/src/app/views/workspace/component.ts [hunk 2]
```diff
@@ -50,6 +51,7 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
   bugReportOpen: boolean;
   bugReportDescription = '';
   googleBucketModal = false;
+  showUpdatedResearchPurpose = environment.enableDatasetBuilder;
```
Comment (gjuggler) [inline ui/src/app/views/workspace/component.ts]: The tying together of these features is a bit surprising – your offline explanation makes total sense, but the code here might benefit from a quick comment (and maybe a link to the dataset builder Epic) saying why we're tying these together and when it should be removed.

File: ui/src/app/views/workspace-about.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        <div style={styles.mainHeader}>Research Purpose
+          <Clickable disabled={!workspacePermissions.canWrite}
+                     data-test-id='edit-workspace'
+                     onClick={() => navigate(
+                       ['workspaces',  workspace.namespace, workspace.id, 'edit'])}>
+            <EditComponentReact disabled={!workspacePermissions.canWrite}
+                                style={{marginTop: '0.1rem'}}/>
+          </Clickable>
+        </div>
+        <div style={styles.sectionText}>{ResearchPurposeDescription}</div>
```
Comment (gjuggler) [inline ui/src/app/views/workspace-about.tsx]: I know Lou's mocks included this, but it feels kind of awkward to have such a long piece of boilerplate text being shown to all users who ever visit any workspace. Maybe it's worth quickly checking w/ Karthik or Lou on whether this is really the intent?

If we do need to keep this in, I'd argue it should be towards the bottom, so more useful content (e.g. the workspace description) is above the fold.

File: ui/src/app/views/workspace/component.ts (hunk 2 above)
Comment (jacmrob) [reply ui/src/app/views/workspace/component.ts]: will add a comment

File: ui/src/app/views/workspace-about.tsx (hunk 3 above)
Comment (jacmrob) [reply ui/src/app/views/workspace-about.tsx]: I totally agree :/ will check with karthik

**Pull request merged**